### PR TITLE
Fix/darkmode bugs

### PR DESCRIPTION
**Pull request by RobinStut**

### CODE REVIEW

File: src/facebook-desktop-preview/facebook-desktop-preview.ejs [hunk 1]
```diff
@@ -350,6 +354,10 @@
       }
     }
 
+    .-theme-with-dark-background {
```
Comment (Maikxx) [inline src/facebook-desktop-preview/facebook-desktop-preview.ejs]: Does this need to be here twice?

File: src/facebook-desktop-preview/facebook-desktop-preview.js [hunk 2]
```diff
@@ -59,7 +61,7 @@ function getfacebookMarkup({ title, description, img, url, desktopImgIsBig, imag
         </div>
       </div>
     </a>
-    <div class="facebook-preview__interaction-wrapper">
+    <div class="facebook-preview__interaction-wrapper  ${theme}">
```
Comment (Maikxx) [inline src/facebook-desktop-preview/facebook-desktop-preview.js]: A space too much here?

File: src/facebook-mobile-preview/facebook-mobile-preview.ejs [hunk 3]
```diff
@@ -255,6 +255,10 @@
       line-height: 20px;
     }
 
+    .-theme-with-dark-background {
+      color: var(--base-color);
```
Comment (Maikxx) [inline src/facebook-mobile-preview/facebook-mobile-preview.ejs:259]: Should there not be a fallback color, for the case css variables are not supported?

File: src/linkedin-preview/linkedin-preview.html [hunk 4]
```diff
@@ -270,6 +270,15 @@
       padding-bottom: 0;
     }
 
+    .linkedin-preview__like-count {
+      color: var(--base-color);
+    }
+
+    .-theme-with-dark-background.linkedin-preview__like-count {
+      color: var(--base-color);
+
```
Comment (Maikxx) [inline src/linkedin-preview/linkedin-preview.html]: Enter too much here

**Review by Maikxx — CHANGES_REQUESTED** [review]

File: src/facebook-mobile-preview/facebook-mobile-preview.ejs (hunk 3 above)
Comment (RobinStut) [reply src/facebook-mobile-preview/facebook-mobile-preview.ejs:259]: I think that kills the idea of having flexible variable which we can change in the future. 
you mean like this right?:
`.-theme-with-dark-background {
      color:#fff;
      color: var(--base-color);
    }`

**Review by Maikxx — APPROVED** [review]

**Pull request merged**